src/Assignments/ProblemIframe.tsx

```diff
@@ -118,13 +110,21 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
                     showSolutions: previewShowSolutions
                 });
             } else {
-                res = await AxiosRequest.get(`/courses/question/${problem.id}${queryString}`);
+                res = await getQuestion({
+                    id: problem.id,
+                    workbookId,
+                    readonly
+                });
             }
             return res.data.data.rendererData.renderedHTML as string;
 
         } catch (e) {
             setError(e.message);
-            logger.error('Error posting preview', e, e.message);
+            
+            if (_.isNil((e as BackendAPIError).status) || ((e as BackendAPIError).status !== 200 && (e as BackendAPIError).status !== 400)) {
```

Comment:
Why don't we do instance checking here so we don't have to keep casting?
You could add this method to BackendAPIError

```Typescript
    static isBackendAPIError(obj: unknown): obj is BackendAPIError {
        return obj instanceof BackendAPIError;
    }
```

or you could just do the instance check here